tests/test_asset_list.py

```diff
@@ -147,6 +148,42 @@ def test_get_field_table_asset_default(self):
         field_table_asset = get_field_table_asset(config)
         self.assertEqual(field_table_asset, DEFAULT_FIELD_TABLE_ASSET)
 
+    def test_get_field_table_asset_existing_filename(self):
+        config = fv3config.get_default_config()
+        with tempfile.NamedTemporaryFile() as field_table:
+            config["field_table"] = field_table.name
+            field_table_asset = get_field_table_asset(config)
+            directory, filename = os.path.split(field_table.name)
+            expected_field_table_asset = get_asset_dict(
+                directory, filename, target_name="field_table"
+            )
+            self.assertEqual(field_table_asset, expected_field_table_asset)
+
+    def test_get_field_table_asset_non_existent_filename_or_directory(self):
+        config = fv3config.get_default_config()
+        config["field_table"] = "foo"
+        with self.assertRaises(fv3config.ConfigError):
+            get_field_table_asset(config)
```

Comment:
Add a case where the non-existent filename is a non-existent absolute path (e.g. `/foo`). I think right now that test would fail. Apart from that distinction you covered all the cases, great job!